spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/converter/PubSubMessageConversionException.java

```diff
@@ -14,14 +14,21 @@
  *  limitations under the License.
  */
 
-package org.springframework.cloud.gcp.pubsub.core.test.disallowed;
+package org.springframework.cloud.gcp.pubsub.support.converter;
+
+import org.springframework.core.NestedRuntimeException;
 
 /**
- * @author Chengyuan Zhao
+ * An exception raised by {@link PubSubMessageConverter} implementations.
```

Comment:
Let's don't repeat an error from the `spring-messaging`!
I mean let's don't mention `PubSubMessageConverter` at all since such an exception can be used in any conversion place, where Pub/Sub is involved.

That's why I would be good to still reuse that `MessageConversionException` in our classes, ever if that has such a misleading JavaDoc.